packages/kbn-tinymath/src/index.js

```diff
@@ -23,7 +24,7 @@ function parse(input, options) {
   }
 
   try {
-    return parseFn(input, options);
+    return memoizeOne(parseFn)(input, options);
```

Comment:
Didn't test (I can do tomorrow), but is this actually memoizing? Looking at the memoize-one source code, it seems like `memoizeOne` itself is not memoized on the passed-in function so it would create a new memoization closure on each call without actually ever hitting the cache.

Reply:
You're totally right, the memoizeOne function returns a instance each time!